libs/librepcb/libraryeditor/newelementwizard/newelementwizardpage_entermetadata.ui

```diff
@@ -173,6 +189,15 @@
    </item>
   </layout>
  </widget>
- <resources/>
+ <resources>
+  <include location="../../../../img/images.qrc"/>
+  <include location="../../../../img/images.qrc"/>
+  <include location="../../../../img/images.qrc"/>
+  <include location="../../../../img/images.qrc"/>
+  <include location="../../../../img/images.qrc"/>
+  <include location="../../../../img/images.qrc"/>
+  <include location="../../../../img/images.qrc"/>
+  <include location="../../../../img/images.qrc"/>
+ </resources>
```

Comment:
I have no idea why QtCreator sometimes adds this crap, but I usually discard these changes to avoid noise in the commits:

```suggestion
 <resources/>
```

Reply:
Ok, I'll change that. It's quite annoying with the additional resource crap, but it seems to be only added once in this quantity (as of now :( )